src/hooks/UrqlUseQuery.re

```diff
@@ -1,13 +1,20 @@
-type executeQueryJs =
-  option(UrqlClientTypes.PartialOperationContextJs.t) => unit;
+type executeQueryJs = UrqlOperations.partialOperationContext => unit;
 
 type useQueryResponseJs('extensions) = (
   UrqlTypes.jsHookResponse(Js.Json.t, 'extensions),
   executeQueryJs,
 );
 
 type executeQuery =
-  (~context: UrqlClientTypes.partialOperationContext=?, unit) => unit;
+  (
+    ~fetchOptions: Fetch.requestInit=?,
+    ~requestPolicy: UrqlTypes.requestPolicy=?,
+    ~url: string=?,
+    ~meta: UrqlOperations.operationDebugMeta=?,
+    ~pollInterval: int=?,
+    unit
+  ) =>
+  unit;
```

Comment:
A good example of the new API where each key on a `partialOperationContext` is a labelled argument that can be provided to the `execute*` function.